spec/api-app-spec.js

```diff
@@ -122,6 +122,17 @@ describe('app module', () => {
     })
   })
 
+  describe('app.getLocaleCountryCode()', () => {
+    it('should be empty or have length of two', () => {
+      let expectedLength = 2;
```

Comment:
Electron's [JS codestyle](https://github.com/electron/electron/blob/master/docs/development/coding-style.md#javascript) doesn't allow semicolons, please remove the semicolon to fix [the lint job](https://github.visualstudio.com/electron/_build/results?buildId=20907&view=logs).